ext/intl/timezone/timezone.stub.php

```diff
@@ -29,32 +29,32 @@ public static function createEnumeration($countryOrRawOffset = null) {}
      * @return IntlTimeZone|null
      * @alias intltz_create_time_zone
      */
-    public static function createTimeZone(string $zoneId) {}
+    public static function createTimeZone(string $timezoneId) {}
 
     /**
      * @return IntlIterator|false
      * @alias intltz_create_time_zone_id_enumeration
      */
-    public static function createTimeZoneIDEnumeration(int $zoneType, ?string $region = null, ?int $rawOffset = null) {}
+    public static function createTimeZoneIDEnumeration(int $type, ?string $region = null, ?int $rawOffset = null) {}
 
     /**
      * @return IntlTimeZone|null
      * @alias intltz_from_date_time_zone
      */
-    public static function fromDateTimeZone(DateTimeZone $zone) {}
+    public static function fromDateTimeZone(DateTimeZone $timezone) {}
 
     /**
-     * @param bool $isSystemID
+     * @param bool $systemId
      * @return string|false
      * @alias intltz_get_canonical_id
      */
-    public static function getCanonicalID(string $zoneId, &$isSystemID = null) {}
+    public static function getCanonicalID(string $timezoneId, &$systemId = null) {}
```

Comment:
I would keep `$isSystemId` here, it's not obvious that `$systemId` is a boolean parameter.

Reply:
Yes, I was hesitant to remove the prefix for a while, so I'm ok with reverting this change.